plugins/providers/docker/driver.rb

```diff
@@ -42,6 +52,16 @@ def build(dir, **opts, &block)
         matches[:id]
       end
 
+      # Check if podman emulating docker CLI is enabled.
+      #
+      # @return [Bool]
+      def podman?
+        execute('docker', '-v')
+          .scan(/podman/)
+          .any?
```

Comment:
This can be reduced down to: `execute("docker", "-v").include?("podman")`

Reply:
Ah, thanks :)